tests/TestOne/buildtool/bin/buildtool

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh -e -x
+#
+# This is a very simple pjc-aware ocamlfind-based build tool.
+
+export OCAMLFIND_CONF=$cur_target_dir/findlib.conf
+
+# Build
+
+DEPENDS=""
+for p in $cur_depends; do
+  if [ "$p" != "buildtool" ]; then
+    DEPENDS="$DEPENDS $p"
```

Comment:
We filter out `buildtool` itself here but `buildtool` should be modelled as `buildTimeDependency` instead.